src/constants.js

```diff
@@ -0,0 +1,57 @@
+const LETTER_SCORES = {
+  A: 1,
+  B: 3,
+  C: 3,
+  D: 2,
+  E: 1,
+  F: 4,
+  G: 2,
+  H: 4,
+  I: 1,
+  J: 8,
+  K: 5,
+  L: 1,
+  M: 3,
+  N: 1,
+  O: 1,
+  P: 3,
+  Q: 10,
+  R: 1,
+  S: 1,
+  T: 1,
+  U: 1,
+  V: 4,
+  W: 4,
+  X: 8,
+  Y: 4,
+  Z: 10
+}
+
+const INCORRECT_TOKENS = ['|', '(', ')'] // I was happy just ignoring them and scoring dog without giving 0 :(
+
+const MULTIPLE = 1
+
+const MULTIPLIER_USAGE = {
+  '{': 0,
+  '}': 0,
+  '[': 0,
+  ']': 0
+}
+
+const SCORE_MULTIPLIER = {
+  '{': 2,
+  '}': 1 / 2,
+  '[': 3,
+  ']': 1 / 3
+}
+
+const VALID_SCORE = true
```

Comment:
I see we have a man of culture 😂 I'm sure your code wouldn't let me feed it "inval}id{word", very good and beyond what was outlined!

Reply:
No I never actually took into account backwards brackets actually 😱.
As the code currently is `'id'` would be calculated at 1/2 multiplier score and the rest will just have a multiplier value of 1.